compiler/rustc_hir_analysis/src/check/compare_impl_item.rs

```diff
@@ -1850,6 +1865,46 @@ pub(super) fn compare_impl_const_raw(
         return Err(diag.emit());
     };
 
+    let impl_ct_predicates = tcx.predicates_of(impl_ct.def_id);
+    let trait_ct_predicates = tcx.predicates_of(trait_ct.def_id);
+
+    // FIXME(generic_consts): Is this correct? We already have an ocx...
+    check_region_bounds_on_impl_item(tcx, impl_ct, trait_ct, false)?;
+
+    // FIXME(generic_consts): It'd be nice if we added a fast-path if the own bounds are empty
```

Comment:
I don't think we need to make this optimal -- things in `compare_impl_item` should prefer correctness, it's just O(# of consts in crate).